crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -89,6 +89,38 @@ impl Default for TypescriptTransformOptionsVc {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, TraceRawVcs, Serialize, Deserialize)]
+pub enum EmotionLabelKind {
+    DevOnly,
+    Always,
+    Never,
+}
+
+//[TODO]: need to support importmap, there are type mismatch between
+//[TODO]: next.config.js to swc's emotion options
+#[turbo_tasks::value(shared)]
+#[derive(Default, Clone, Debug)]
+pub struct EmotionTransformOptions {
+    pub enabled: bool,
```

Comment:
Do we need an `enabled` if the `enable_emotion` if this is only used inside an `Option` wrapper? Shouldn't the `Some(EmotionTransformOptions)` imply that it is enabled?

Reply:
That's how I started, and then ends up this was somewhat more ergonomics when try to read next.config.js, we have couple of conditions

1. if compiler option non-exists - None
2. if exists, somehow user configures `emotion: false` or `emotion: {enabled :false}`

and having Option<T> was easier to gracefully read & create config value among those conditions. It is possible to argue who would do 2 (if you specifiy compiler option, explicitly disabling it is non common) but that's certainly possible case if someone relies on some dynamic evaluation based on env / fn.